pkg/services/sso/redhatsso_service.go

```diff
@@ -29,7 +69,7 @@ func (r *redhatssoService) DeRegisterClientInSSO(accessToken string, clientId st
 	glog.V(5).Infof("Deregistering client with id: %s", clientId)
 	err := r.client.DeleteServiceAccount(accessToken, clientId)
 	if err != nil {
-		return errors.NewWithCause(errors.ErrorFailedToDeleteSSOClient, err, "failed to delete the sso client")
+		return errors.NewWithCause(errors.ErrorFailedToDeleteSSOClient, err, getErrorDescription(err, "failed to delete the sso client"))
 	}
 	glog.V(5).Infof("Deregistered client with id: %s", clientId)
```

Comment:
getErrorDescription can also be added in the logs this will help to debug issues